resources/coaching-advice.md

```diff
@@ -0,0 +1,63 @@
+
+---
+
+There is a fundamental difference between a **coach** and a **teacher**. The goal isn't to stand in front of a class and lecture them about Rust but to stand *besides* them and helping them discover the language by themselves. Some things to consider are specific to teaching programming (or other tech related fields) and having experience with that is always a plus.
+
+- Stand on the sidelines but be ready to help
+- Focus on the students
+- Have sympathy and patience for their (in-)abilities
+- Be positive and encouraging
+  - It's okay to make mistakes
+  - And take breaks if they get frustrated!
+  - Leave time for experimentation and "playing around"
+  - If people are getting "off the path", don't force them back on!
+- Make sure people are having fun
+
+### General teaching advice
+
+Others are generally good advice when teaching anything (even in a more hands-off approach)
+
+- Create a friendly and welcoming environment
+  - Use students names and pronouns
+  - Smile at them, make eye contact
+- Admit if you don't know something; that's okay!
+- When explaining, try to avoid jargon, especially on tech subjects
+  - Don't assume pre-existing knowledge
+  - Try to rephrase things in case of misunderstandings
+- Don't accept students self-deprecation
+  - Tell them that they can do it
+  - Congratulate them on achievements
+  - Let people show off their work if they want to
+  
+### Dealing with questions
+  
+Be sure to be engaged with the students problems. Just because someone isn't asking questions that doesn't mean they're not having problems. 
+
+- They might be too afraid to ask.
+- Check in with people once in a while and ask "Hey, how is it going? Anything I can help you with?"
+  - This helps shy learners, builds connection and increass engagement
+  - Alternatively: Sit next to them and talk about what they are doing.
+- Generally encourage the asking of questions
+  - Building the right atmosphere is important but also sometimes difficult
+  - Emphasize that there's no such thing as a "dumb" question
+- Positively respond to questions
+- If a lot of people are struggling with a certain subject, be sure to adjust your material
+
+### Pacing
+
+- This isn't about you, it's about the students; go at their pace
+- Talk slowly, leave space after saying something
+- When answering questions be sure to leave them time to think about it. Then make sure they understood it
+
+
+### **Don'ts**
+
+- Follow the code of conduct of your event, **be an example that students can follow!**
+  - This means no harassment, no sexual advances, no slurs or violating participants boundries
```

Comment:
additionally, if you see any of your fellow teachers/mentors/organizers doing this, confront them immediately